File: snippets/fsharp/System/UInt16/MaxValue/MaxValue.fs
// <Snippet1>
open System

let integerValue = 1216

if integerValue >= int UInt16.MinValue && integerValue <= int UInt16.MaxValue then
    let uIntegerValue = uint16 integerValue
    printfn $"{uIntegerValue}"
else
    printfn $"Unable to convert {integerValue} to a UInt16t."
// </Snippet1>

File: snippets/csharp/System/UInt16/MaxValue/MaxValue.cs
using System;

public class Class1
{
    public static void Main()
    {
        // <Snippet1>
        int integerValue = 1216;
        ushort uIntegerValue;

        if (integerValue >= ushort.MinValue & integerValue <= ushort.MaxValue)
        {
            uIntegerValue = (ushort)integerValue;
            Console.WriteLine(uIntegerValue);
        }
        else
        {
            Console.WriteLine($"Unable to convert {integerValue} to a UInt16t.");
        }
        // </Snippet1>
    }
}
